{
  "networkCallRecords" : [ {
    "Method" : "POST",
    "Uri" : "https://REDACTED.vault.azure.net/certificates/restore?api-version=7.4",
    "Headers" : {
      "User-Agent" : "azsdk-java-client_name/client_version (18.0.2; Windows 11; 10.0)",
      "Content-Type" : "application/json"
    },
    "Response" : {
      "X-Content-Type-Options" : "nosniff",
      "Pragma" : "no-cache",
      "retry-after" : "0",
      "StatusCode" : "400",
      "Date" : "Thu, 16 Mar 2023 08:09:13 GMT",
      "Strict-Transport-Security" : "max-age=31536000;includeSubDomains",
      "Cache-Control" : "no-cache",
      "x-ms-keyvault-network-info" : "conn_type=Ipv4;addr=187.190.190.225;act_addr_fam=InterNetwork;",
      "x-ms-keyvault-region" : "westus",
      "Expires" : "-1",
      "Content-Length" : "78",
      "x-ms-keyvault-service-version" : "1.9.736.1",
      "x-ms-request-id" : "04ed163e-8f96-4e51-9232-efb45942094b",
      "Body" : "{\"error\":{\"code\":\"Malformed backup blob\",\"message\":\"Backup blob is corrupt.\"}}",
      "Content-Type" : "application/json; charset=utf-8"
    },
    "Exception" : null
  } ],
  "variables" : [ ]
}